### PR TITLE
adds visual representation of snowman and melting effect

### PR DESCRIPTION
**Pull request by NicoleCarpenter**

### CODE REVIEW

**Comment by NicoleCarpenter** [comment]
@mnewsome @zacharyjdavy

**Comment by zdavy** [comment]
:+1: on this, except wait until after the IPM to merge this one, I want to see what the user experience is like first.

File: Sources/Application/View/images/SnowManImage.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "         |      |          ",
+  "      <============>       ",
+  "        /       \\         ",
+  "       |   * *   |         ",
+  "       |    =>   |         ",
+  "        \\ `---` /         ",
+  " \\|/     -------     \\|/ ",
+  "  \\    /    *    \\    /  ",
+  "   \\===     *     ===/    ",
+  "      |     *     |        ",
```
Comment (mnewsome) [inline Sources/Application/View/images/SnowManImage.swift:13]: Is his button off center?

**Comment by mnewsome** [comment]
:+1:  Just one question to make sure the snowman's buttons are lined up

File: Sources/Application/View/images/SnowManImage.swift (hunk 1 above)
Comment (NicoleCarpenter) [inline Sources/Application/View/images/SnowManImage.swift:13]: No, that is just because of the escaped backslash

**Pull request merged**